modules/logger/logger.py

```diff
@@ -143,6 +145,48 @@ def info(self, message: str, log_with_frame_info: bool = True) -> None:
             message = self.message_and_metadata(message, caller_frame)
         self.logger.info(message)
 
+    def save_image(
```

Comment:
Also, can you move this function to the bottom (like after all the other log text things functions?)